src/NovelRT.Interop/Audio/NrtAudioService.cpp

```diff
@@ -160,7 +252,16 @@ NrtResult Nrt_AudioService_playSound(NrtAudioService service, unsigned int handl
     }
 
     auto serv = reinterpret_cast<Audio::AudioService*>(service);
-    serv->playSound(handle, loops);
+
+    try
+    {
+      serv->playSound(handle, loops);
+    }
+    catch (const Exceptions::NotInitialisedException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?